apps/pie-docs/src/src.11tydata.js

```diff
@@ -0,0 +1,26 @@
+const projectSettings = require('./_data/projectSettings');
+
+// Only build drafts when site is in development mode
+const shouldBuildDrafts = projectSettings().environment === 'development';
+
+module.exports = {
+    eleventyComputed: {
+        permalink (data) {
```

Comment:
maybe worth adding a comment explaining that the absence of a permalink results in 11ty ignoring/not building the file?